templates/incl/menubar.html

```diff
@@ -34,6 +34,7 @@
                 <a class="dropdown-item" href="/highscores"><span class="fas fa-trophy ltr:mr-4 rtl:ml-4"></span>{{_('highscores')}}</a>
                 <a class="dropdown-item" href="/my-achievements"><span class="fas fa-fw fa-list-alt ltr:mr-4 rtl:ml-4"></span>{{_('my_achievements')}}</a>
                 <a class="dropdown-item" href="/my-profile"><span class="fas fa-user-cog ltr:mr-4 rtl:ml-4"></span>{{_('my_account')}} {% if get_user_messages() %} ({{ get_user_messages() }}){% endif %}</a>
+                <a class="dropdown-item" id="public_profile_redirect" href="#" onclick="window.open('/user/{{ username}}')"><span class="fas fa-user-cog ltr:mr-4 rtl:ml-4"></span>{{_('visit_own_public_profile')}} {% if not public_settings %}hidden{% endif %}</a>
```

Comment:
I think there's no need to show this link if the user doesn't have a public profie, can you add an if that only shows this if that's the case?

Reply:
Also, since you used the same babel label here as in the button, it says "Visit your own profile", wich is too long for this menu. Could you add a new lable that says "My public profile". For this you'd need to add a new Babel label.

Reply:
Hi, I fixed it, but I'm not sure how to add Babel label.